src/runtime/virtcontainers/pkg/agent/protocols/client/client.go

```diff
@@ -425,9 +428,16 @@ func HybridVSockDialer(sock string, timeout time.Duration) (net.Conn, error) {
 		case err = <-errChan:
 			if err != nil {
 				conn.Close()
-				agentClientLog.WithField("Error", err).Debug("HybridVsock trivial handshake failed")
-				return nil, err
 
+				// With full debug logging enabled there might be around 1,500 redials in a tight loop, so
+				// skip logging some of these failures to avoid flooding the log.
+				errorsCount := hybridVSockErrors
```

Comment:
For my learning, why can't we just use hybridVSockErrors and do 
hybridVSockErrors =  hybridVSockErrors + 1
on L438

Reply:
Using a separate errorsCount variable made this code easier to read/understand for me. Otherwise, I agree that most likely the same code gets generated when using hybridVSockErrors directly.

Reply:
I would probably agree with Saul and just used a single variable here, but it's not a significant problem for me, it just took a bit more brain power to check there wasn't something extra going on I was missing 😄